akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -620,7 +627,7 @@ final class ClusterReceptionist(pubSubMediator: ActorRef, settings: ClusterRecep
       pubSubMediator.tell(msg, tunnel)
 
     case Heartbeat ⇒
-      log.debug("Heartbeat from client [{}]", sender().path)
+      if (verboseHeartbeatLogging) log.debug("Heartbeat from client [{}]", sender().path)
```

Comment:
Right, I was thinking if we want to silence both actually. Sounds good, I'll do that